src/ui/room_properties.ui

```diff
@@ -0,0 +1,338 @@
+     </property>
+     <property name="sizeHint" stdset="0">
+      <size>
+       <width>0</width>
+       <height>0</height>
+      </size>
+     </property>
+    </spacer>
+   </item>
+   <item row="2" column="0" colspan="2">
+    <widget class="QGroupBox" name="groupBox_pathfinding">
+     <property name="title">
+      <string>Pathfinding</string>
+     </property>
+     <layout class="QGridLayout" name="gridLayout_9">
+      <item row="1" column="0" colspan="2">
+       <layout class="QHBoxLayout" name="horizontalLayout_3">
+        <item>
+         <widget class="QSpinBox" name="spinBox_weight">
+          <property name="sizePolicy">
+           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
+            <horstretch>0</horstretch>
+            <verstretch>0</verstretch>
+           </sizepolicy>
+          </property>
+          <property name="minimum">
+           <number>1</number>
+          </property>
+          <property name="maximum">
+           <number>2147483647</number>
+          </property>
+          <property name="value">
+           <number>1</number>
+          </property>
+         </widget>
+        </item>
+        <item>
+         <widget class="QComboBox" name="comboBox_weight">
+          <property name="sizePolicy">
+           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
+            <horstretch>0</horstretch>
+            <verstretch>0</verstretch>
+           </sizepolicy>
+          </property>
+          <property name="editable">
+           <bool>true</bool>
+          </property>
+         </widget>
+        </item>
+       </layout>
+      </item>
+      <item row="0" column="0" colspan="2">
+       <widget class="QLabel" name="label_weightInstructions">
+        <property name="text">
+         <string>Enter new roomweight (= travel time), minimum (and default) is 1:</string>
+        </property>
+       </widget>
+      </item>
+      <item row="3" column="0">
+       <widget class="QCheckBox" name="checkBox_locked">
+        <property name="text">
+         <string>Lock room, so it will never be used for speedwalking</string>
+        </property>
+       </widget>
+      </item>
+     </layout>
+    </widget>
+   </item>
+   <item row="1" column="0" colspan="2">
+    <widget class="QGroupBox" name="groupBox_layout">
+     <property name="title">
+      <string>Symbol</string>
+     </property>
+     <layout class="QGridLayout" name="gridLayout_8">
+      <item row="0" column="0" colspan="4">
+       <widget class="QLabel" name="label_symbolInstructions">
+        <property name="text">
+         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Choose: &lt;br/&gt;• an existing symbol from the list below (sorted by most commonly used first)&lt;br/&gt;• enter one or more graphemes (&amp;quot;visible characters&amp;quot;) as a new symbol&lt;br/&gt;• enter a space to clear any existing symbols&lt;br/&gt;for all of the 5 selected rooms:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```

Comment:
Common folk don't know what a grapheme is - I recommend just "one or more characters".

Reply:
This was inspired by the previous text. Trying to change as few as possible but just making it a bit more concise. Let's see if @SlySven agrees to your proposal as well. Looks like a great opportunity for bike shedding maybe..

Reply:
We need to be as precise as possible because it can get very confusing to differentiate between *grapheme* and *character* and *symbol* here. A room symbol in Mudlet is technically a "sequence of one or more graphemes" and as far as our symbol handling code goes each distinct, unique  "sequence of one or more graphemes" is a different "room symbol" but each one can be made from a series of Unicode code points or "characters" as can be seen in the "Map symbol usage" dialogue. Also, as we all know, one code point is NOT necessarily one character nor is it necessarily one grapheme.
![image](https://user-images.githubusercontent.com/6163092/199851703-b18d58c1-6003-448f-9c1f-ceb5f3eaac6d.png)

Reply:
I also found the name "glyphs" used in another place in Mudlet. This seems to warrant more discussion .. elsewhere!

Reply:
@SlySven we do not "all" know! The vast majority of players do not, and technically complex language like this will make players feel Mudlet is not designed for them, but for experts. Let's have this discussion elsewhere though as suggested, the language was already in.